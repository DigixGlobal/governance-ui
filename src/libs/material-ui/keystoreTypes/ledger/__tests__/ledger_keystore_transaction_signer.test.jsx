import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import LedgerKeystoreTransactionSigner from '../ledger_keystore_transaction_signer';

const address = {
  kdPath: 'kdPath test',
  address: '0x123435',
};

describe('<LedgerKeystoreTransactionSigner />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <LedgerKeystoreTransactionSigner
        hideTxSigningModal={jest.fn}
        address={address}
        txData={{}}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
