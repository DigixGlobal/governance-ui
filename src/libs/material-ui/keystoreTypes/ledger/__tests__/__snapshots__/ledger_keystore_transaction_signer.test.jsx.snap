// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LedgerKeystoreTransactionSigner /> renders correctly 1`] = `
<LedgerContianer
  expect={
    Object {
      "address": "0x123435",
      "kdPath": "kdPath test",
    }
  }
  onReady={[Function]}
  renderReady={[Function]}
/>
`;
